.lintstagedrc.mjs

```diff
@@ -2,6 +2,6 @@ export default {
     '*': 'prettier --write --ignore-unknown',
     'packages/**/*.{ts,tsx,js,jsx}': 'eslint -c packages/.eslintrc.json --fix',
     'packages/mask/**/*': () => 'gulp locale-kit --sync-keys --remove-unused-keys',
-    'packages/web3-constants/**/*': 'pnpm start --filter web3-constants',
+    'packages/web3-constants/**/*': 'pnpm --filter ./packages/web3-constants start',
```

Comment:
```suggestion
    'packages/web3-constants/**/*': () => 'pnpm --filter ./packages/web3-constants start',
```

reduce file list input on the command

Reply:
Sorry, but what's the difference?

Reply:
lint-staged default inject file list into command arguments list tail

https://github.com/okonet/lint-staged#example-run-tsc-on-changes-to-typescript-files-but-do-not-pass-any-filename-arguments